src/routes/[id]/+page.svelte

```diff
@@ -68,81 +73,79 @@
   }
 </script>
 
-
 <svelte:head>
   {@html atomOneDark}
 </svelte:head>
 
-{#if paste}
-  {handleassignment()}
+{#if errorMessage}
+  <div class="flex justify-center items-center h-screen bg-gray-900">
+    <div
+      class="max-w-lg w-full mx-auto bg-red-900 text-white rounded-lg shadow-xl p-8"
+    >
+      <div class="flex items-center space-x-3 mb-4">
+        <svg
+          class="w-8 h-8 text-red-300"
+          fill="currentColor"
+          viewBox="0 0 20 20"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            fill-rule="evenodd"
+            d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-7 4a1 1 0 10-2 0 1 1 0 002 0zm-.707-7.707a1 1 0 00-1.414 0L8.293 7.293a1 1 0 001.414 1.414L10 7.414l1.293 1.293a1 1 0 001.414-1.414l-2-2z"
+            clip-rule="evenodd"
+          ></path>
+        </svg>
+        <h2 class="text-xl font-bold">Error</h2>
+      </div>
+      <p class="text-md mb-4">{errorMessage}</p>
+      <div class="flex justify-end">
+        <button
+          on:click={() => goto("/")}
+          class="bg-red-700 hover:bg-red-600 text-white py-2 px-4 rounded-md focus:outline-none focus:ring-2 focus:ring-red-500"
+        >
+          Go Back
+        </button>
+      </div>
+    </div>
+  </div>
+{:else if paste}
   <div class="container mx-auto my-8">
     <div class="bg-gray-800 rounded-lg shadow-md p-6">
       <div class="card-actions">
-        <div
-          class="badge badge-outline flex justify-betweenflex justify-between"
-        >
-          <h1 class="border rounded-lg p-2 text-white text-2xl mb-4">{paste.title}</h1>
-         
+        <div class="badge badge-outline flex justify-between">
+          <h1 class="border rounded-lg p-2 text-white text-2xl mb-4">
+            {paste.title}
+          </h1>
         </div>
 
-        <HighlightAuto {code} />
+        <HighlightAuto code={paste.text} />
       </div>
       <Toaster />
       <div class="flex justify-between">
-        <!-- copy button -->
         <button
           on:click={() => {
-            // toast("Here is your toast.");
-            toast.success("copied to clipboard");
+            toast.success("Copied to clipboard");
           }}
           class="btn-clip bg-green-600 hover:bg-lime-600 text-white font-medium py-2 px-4 mt-3 rounded-md focus:outline-none focus:ring-2 focus:ring-indigo-500"
-          data-clipboard-text={code}
+          data-clipboard-text={paste.text}
         >
           COPY TEXT
         </button>
         <div>
-          <!-- download pdf button -->
           <button
```

Comment:
Use old button theme for download and share 

![image](https://github.com/user-attachments/assets/52bc02c0-96cc-470f-8eb0-08a9954da6b1)